migration-guide/src/explorer-api.md

```diff
@@ -0,0 +1,2904 @@
+              "getCoin": "0"
+            }
+          },
+          {
+            "cbeEpoch": 106,
+            "cbeSlot": 306880,
+            "cbeBlkHeight": 2210372,
+            "cbeBlkHash": "904f180c9271d96b6086623daa402e9f033a05ecf1bf35f9f5508b4584b6fed8",
+            "cbeTimeIssued": 1610098496,
+            "cbeTxNum": 0,
+            "cbeTotalSent": {
+              "getCoin": "0"
+            },
+            "cbeSize": 3,
+            "cbeBlockLead": "6df3e1b4b8a84c63c805076a85e5aa00924997a4eae85fddf0aee3ca",
+            "cbeFees": {
+              "getCoin": "0"
+            }
+          },
+          {
+            "cbeEpoch": 106,
+            "cbeSlot": 306800,
+            "cbeBlkHeight": 2210371,
+            "cbeBlkHash": "8c262185505782726c09d68cb0b84aab12f84117cb554b25ab2212d69eaed083",
+            "cbeTimeIssued": 1610098416,
+            "cbeTxNum": 0,
+            "cbeTotalSent": {
+              "getCoin": "0"
+            },
+            "cbeSize": 3,
+            "cbeBlockLead": "b3873a254459f506e47b9a252ee7912e538b364447f31576a170db65",
+            "cbeFees": {
+              "getCoin": "0"
+            }
+          }
+        ]
+      ]
+    }
+    ```
+    </details>
+
+=== "cardano-graphql"
+
+    **query.graphql**
+    ```graphql
+    query getBlocks($limit: Int!, $epoch: Int!) {
+      blocks(limit: $limit, where: { epoch: { number: { _eq: $epoch } } }) {
+        slotNo # cbeSlot
+        slotInEpoch # cbeEpoch
+        number # cbeBlkHeight
+        hash # cbeBlkHash
+        forgedAt # cbeTimeIssued
+        transactionsCount # cbeTxNum
+        transactions_aggregate {
+          aggregate {
+            sum {
+              totalOutput # cbeTotalSent
+            }
+          }
+        }
+        slotLeader {
+          hash # cbeBlockLead
+        }
+        size # cbeSize
+        fees # cbeFees
+      }
+    }
+    ```
+    **variables.json**
+    ```json
+    {
+      "limit": 2,
+      "epoch": 86
+    }
+    ```
+
+    ```console
+    $ curl 'https://cardano-graphql-testnet.daedalus-operations.com/' -H 'Accept-Encoding: gzip, deflate, br' -H 'Content-Type: application/json' -H 'Accept: application/json' -H 'Connection: keep-alive' -H 'DNT: 1' -H 'Origin: https://cardano-graphql-testnet.daedalus-operations.com' --data-binary '{"query":"    query getBlocks($limit: Int!, $epoch: Int!) {\n      blocks(limit: $limit, where: { epoch: { number: { _eq: $epoch } } }) {\n        slotNo # cbeSlot\n        slotInEpoch # cbeEpoch\n        number # cbeBlkHeight\n        hash # cbeBlkHash\n        forgedAt # cbeTimeIssued\n        transactionsCount # cbeTxNum\n        transactions_aggregate {\n          aggregate {\n            sum {\n              totalOutput # cbeTotalSent\n            }\n          }\n        }\n        slotLeader {\n          hash # cbeBlockLead\n        }\n        size # cbeSize\n        fees # cbeFees\n      }\n    }\n","variables":{"limit":2,"epoch":86}}' --compressed
```

Comment:
This is copy pasted from the `COPY CURL` button in graphql playground. I found it too much fiddling dealing with files. It's meant to be copy-pasted to cli, so doesn't really matter that it's a long line.